databuilder/extractor/dashboard/mode_analytics/mode_dashboard_last_successful_executions_extractor.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from pyhocon import ConfigTree, ConfigFactory  # noqa: F401
+
+from databuilder.extractor.dashboard.mode_analytics.mode_dashboard_executions_extractor import \
+    ModeDashboardExecutionsExtractor
+from databuilder.extractor.dashboard.mode_analytics.mode_dashboard_utils import ModeDashboardUtils
+from databuilder.extractor.restapi.rest_api_extractor import STATIC_RECORD_DICT
+from databuilder.models.dashboard.dashboard_execution import DashboardExecution
+from databuilder.rest_api.rest_api_query import RestApiQuery
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ModeDashboardLastSuccessfulExecutionExtractor(ModeDashboardExecutionsExtractor):
```

Comment:
do you think we should add unit test or it is already covered in the parent class test?

Reply:
> do you think we should add unit test or it is already covered in the parent class test?

In my opinion, all is covered by parent class. Let me know.